xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -104,20 +104,23 @@
          */
         private String description;
 
-        ExportArguments(XWikiContext context) throws XWikiException
+        ExportArguments(XWikiContext context, String format) throws XWikiException
         {
             XWikiRequest request = context.getRequest();
 
             this.description = request.get("description");
 
             this.name = request.get("name");
-            if (StringUtils.isBlank(name)) {
-                this.name = context.getDoc().getFullName();
-            }
 
             String[] pages = request.getParameterValues("pages");
             String[] excludes = request.getParameterValues("excludes");
 
+            if (StringUtils.isBlank(name)) {
+                if (!format.equals("xar")) {
```

Comment:
You could have used a single ``if`` (with 2 conditions).

Reply:
indeed